requestium/requestium.py

```diff
@@ -69,7 +71,7 @@ def _start_phantomjs_browser(self):
 
         # Create driver process
         return RequestiumPhantomJS(executable_path=self.webdriver_path,
-                                   service_log_path="/tmp/ghostdriver.log",
+                                   service_log_path=os.path.join(tempfile.tempdir,"ghostdriver.log"),
```

Comment:
This line should be
```
os.path.join(tempfile.gettempdir(), "ghostdriver.log")
```
Instead of using `tempfile.tempdir` directly, right?

Reply:
right

Reply:
If you make the change, it can be merged :)